package.json

```diff
@@ -13,8 +13,6 @@
     "@supabase/supabase-js": "^2.46.1",
     "bootstrap": "^5.3.3",
     "dotenv": "^16.4.5",
```

Comment:
PRAISE: Hooray! Nice job removing unneeded dependencies! Hard to believe it's only been a couple weeks since I even thought about MongoDB!